templates/webapi/test/infopanel.html.ep

```diff
@@ -46,12 +46,14 @@
                     % else {
                         % my $state_detail = '';
                         % if (my @gru_deps = $job->gru_dependencies) {
+                            % my $minion_id = $gru_deps[0] ? $gru_deps[0]->gru_task_id : "";
+                            % my $link = $gru_deps[0] ? url_for('/minion/jobs?note=gru_id_' . $minion_id) : "";
```

Comment:
Please use the `query` method to add query parameters:
```suggestion
                            % my $link = $gru_deps[0] ? url_for('/minion/jobs')->query(note => "gru_id_$minion_id") : "";
```
But I would add the link for every gru task and also add the task name in the query.